packages/@extension-kid/toast-notifications/components/Toast.tsx

```diff
@@ -1,10 +1,20 @@
 import * as React from "react";
+import { Trans } from "@lingui/macro";
 import { Toast } from "@dcos/ui-kit";
 import { ToastProps } from "@dcos/ui-kit/dist/packages/toaster/components/Toast";
 
-import { ToastNotification } from "../ToastNotification";
+import { ToastNotification, ToastTranslatableText } from "../ToastNotification";
 import ToastExtension from "../ToastExtension";
 
+function makeTextContent(
+  value: string | ToastTranslatableText
+): React.ReactElement<HTMLElement> {
+  if (typeof value === "string") {
+    return <Trans id={value} />;
+  }
+  return <Trans id={value.id} values={value.values} />;
```

Comment:
I'm not 100% certain here, but it seems based on the docs that `values` is only a prop on `Trans` from `@lingui/react`? Would it be possible to restrict `value` here to `string | ReactNode` so that it can be either "regular title" or `<Trans>Title formatted in whatever way the user wants so they don't have to know how to format 'values' in this context</Trans>`?

Reply:
Do you mean allowing the value on the ToastNotification class be a ReactNode so we can set it to Trans?

I was trying to avoid that but I’m not completely opposed to it.

Reply:
Yes I guess it would be changing the `title` on ToastNotification as well to `string | ReactNode`. IMO using the `<Trans>` macro is a more familiar pattern than using the bare `<Trans>` @lingui/React component with `values` prop, but it's not a blocker for me

Reply:
I'll try this out today and see what the change would look like.

My two concerns are 1) any file that needs to create and push a notification would then have to import React and be a `tsx` file. Which isn't a deal breaker but its annoying. 2) I'm currently using the content of the title and description to build a hash that is used as the unique id for the toast. This is so that we can ensure previously dismissed toasts aren't re-pushed.